test/bdd/features/vc_e2e_api.feature

```diff
@@ -21,12 +21,17 @@ Feature: Using VC REST API
+      | myprofile_prc           | permanent_resident_card.json |                                                                  |                                                                                          | ProofValue      |          {"driverURL":"http://uni-registrar-web:9080/1.0/register?driver-did-method-rest"}                                                                    | did:trustbloc   |   JsonWebSignature2020  |           |
+      | myprofilewithdidv1      | university_degree.json       |                                                                  |                                                                                          | JWS             |       {"driverURL":"http://uni-registrar-web:9080/1.0/register?driver-universalregistrar/driver-did-v1","options": {"ledger": "test", "keytype": "ed25519"}}  | did:v1:test:nym |    Ed25519Signature2018 |           |
+      | myprofilewithdidelem    | university_degree.json       | did:elem:EiAWdU2yih6NA2IGnLsDhkErZ8aQX6b8yKt7jHMi-ttFdQ          | 5AcDTQT7Cdg1gBvz8PQpnH3xEbLCE1VQxAJV5NjVHvNjsZSfn4NaLZ77mapoi4QwZeBhcAA7MQzaFYkzJLfGjNnR | JWS             |                                                                                                                                                               | did:elem        |    Ed25519Signature2018 |           |
+      | myprofilewithdidsov     | university_degree.json       |                                                                  |                                                                                          | JWS             |       {"driverURL":"https://uniregistrar.io/1.0/register?driver-universalregistrar/driver-did-sov","options": {"network":"danube"}}                           | did:sov:danube  |    Ed25519Signature2018 |           |
+      | myprofile_ud_ed25519    | university_degree.json       |                                                                  |                                                                                          | JWS             |                                                                                                                                                               | did:trustbloc   |   JsonWebSignature2020  |  Ed25519  |
+      | myprofile_ud_p256_jws   | university_degree.json       |                                                                  |                                                                                          | JWS             |                                                                                                                                                               | did:trustbloc   |   JsonWebSignature2020  |  P256     |
```

Comment:
wyu just add P256    to keyType  in myprofile_prc

Reply:
it's already empty

Reply:
That't with uniRegistrar value. It's empty in this profile.